docs/20_smart-contracts/10_getting-started/20_accounts/index.md

```diff
@@ -0,0 +1,208 @@
+
+
+## Public/Private Keys
+
+Every EOS account is ultimately controlled by a public/private key pair. The private key is used to sign transactions and must be kept confidential, while the public key is used to identify the account on the blockchain and can be publicly known.
+
+It is important to safeguard your private key, as it provides complete control over your account's digital assets. Your private key should be stored in a secure software or hardware wallet, as whoever possesses it will have full access to the assets linked to your account.
+
+EOS accounts offer extra security mechanisms out of the box, using something called the *permission system*.
+
+Examples of private and public keys: 
+```
+Private: 5KSdyAiFzYQAtBKDBKCCF28KMMhZ4EmXUxSg8B3nSkHKutT15rY
+Public: PUB_K1_5d7eRKgCCiEdsbBdxxnZdFWnGYS64uWZPZgTcTU1xnB2aESxqR
+Legacy Public Format: EOS5d7eRKgCCiEdsbBdxxnZdFWnGYS64uWZPZgTcTU1xnB2cq4JMD
+```
+
+
+## Permissions system
+
+Each account has a set of hierarchical permissions that control it, and comes with two base permissions by default. These two permissions cannot be removed as they are required for your account to function properly. 
+
+The mandatory permissions are `owner` and `active`.
+
+A permission can only ever change its own information or the information of its children. It can never change its parent’s information.
+
+![Who can change permissions?](images/who_can_change_permissions.png)
+
+
+The information that controls a permission is either another `account@permission` or a public key. This allows for the creation of complex account control structures, where multiple parties can have control over a single account while having full autonomy over their own account’s security. 
+
+Take the following diagram as an example, where the account `alice` is controlled both by `bob` and `charlie`, while `charlie` is also controlled by `tom`. Eventually, all accounts are controlled by public keys. 
+
+
+![Delegated account ownership](images/delegated_account_ownership.png)
+
+
+You can add custom permissions underneath `active` which allows you to limit that permission’s access to only a specific contract’s action (callable function). That permission will then only ever be able to interact with the contract action you specified. 
+
+This means you are able to create granular access permissions across accounts and have hierarchical ownership and usage of them. 
+
+
+![Custom permissions](images/custom_permissions.png)
+
+
+Most importantly, the permission system has built-in support for multi-signature transactions (transactions that require multiple parties to sign them). Every linked account or key associated with a permission has a **weight** assigned to it, and the permission itself has a **threshold**. 
+
+As you can see in the example below, `bob` alone does not have enough power to sign using the `active` permission. He needs either `charlie` or `jenny` to co-sign with him for any transaction that `alice@active` makes. 

```

Comment:
also important to say is that `charlie` and `jenny` cannot sign together, they need `bob`.